index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <title>Real food made Real Good!</title>
+</head>
+<!-- Header Section -->
+<header> Real food made Real Good!</header>
```

Comment:
please use the header h1 element. This ensures good semantic flow for your code and is suitable for screen readers.